decorator.rb

```diff
@@ -0,0 +1,22 @@
+require './nameable'
+
+class Decorator < Nameable
+  attr_accessor :person
+
+  def initialize(person)
+    super()
+    @person = person
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @person.correct_name.capitalize
+  end
+end
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @person.correct_name[0, 10]
+  end
+end
```

Comment:
- [ ] As requested, let's trim the word if the name has a maximum of 10 character

Reply:
Hey @uwadonat what does trim a word mean?

Reply:
> * [ ]  As requested, let's trim the word if the name has a maximum of 10 character

Hi @uwadonat, 
The trimmer decorator class is working just fine and I tested it see the image below.

![image](https://user-images.githubusercontent.com/68796232/186032940-3a73e8ed-aad8-4575-a0ec-dcfeb22c8d3b.png)

Reply:
Hi! Sorry for the delay. It was requested to check for the length of the word first to see if number of character exceed 10 before applying this method.